recipes-openxt/xenclient-root-ro/xenclient-root-ro/init.root-ro

```diff
@@ -223,64 +223,37 @@ fail_first_seal()
 
 seal()
 {
-    local tss_file=${1}
-    local sealerr=0
-    local file_exists=1
+    local config_key="/config/keys/config.key"
+    local sealed_key="$1"
     local sealout=""
-    local seal_file="/config/keys/config.key"
+    local sealrc=""
+    local pcr_opts
+
     maybe_break "seal"
+
     dialog --colors --backtitle "${BACK_TITLE}" --mixedgauge \
         "  Sealing..." 0 0 20
 
-    is_tpm_2_0
-    local tpm2=$?
-
-    if [ "${tpm2}" -eq 0 ];
-    then
-        clean_old_tpm_files
-        if pcr_bank_exists "sha256"; then
-            sealout=$(tpm2_sealdata -H 0x81000000 -I ${seal_file} -O ${tss_file}.sha256 -o ${tss_file}.pub.sha256 -g 0xB -G 0x8 -b 0x492 $(cat /config/config.pcrs) 2>&1 )
-            cp /config/config.pcrs ${tss_file}.pcrs
-            sealerr=$?
-        fi
-    else
-        sealout=$(tpm_sealdata_sa -i /config/keys/config.key \
-            -o ${tss_file} -z $(cat /config/config.pcrs) 2>&1)
-        sealerr=$?
-    fi
-
-    if [ -e ${tss_file} ]; then
-        file_exists=0
-    elif [ -e ${tss_file}.sha256 ]; then
-        file_exists=0
-    else
-        file_exists=1
+    for p in $(sed -e 's/-[rp]//g' "/config/config.pcrs"); do
```

Comment:
I thought you dropped the -r/-p from config.pcrs?  Use `read_pcr_selection`?

Reply:
`/boot/system/tpm/config.tss.pcrs` not longer has the options.
`/config/config.pcrs` still does.

Maybe we should remove that across the board then.

Reply:
Yeah, I would favor just dropping them.